eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -63,15 +64,14 @@ jobs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
 
-      - script: 'dotnet test $(ProjectFile) --logger trx'
+      - script: "dotnet test eng/service.proj --logger trx /p:ServiceDirectory=${{ parameters.ServiceDirectory }}"
         displayName: 'Build & Test (all tests)'
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
           AZURE_TEST_MODE: "None"
-          ${{ each envvar in parameters.envvars }}:
-            ${{ envvar.key }}: ${{ envvar.value }}
+          ${{ insert }}: ${{  parameters.EnvVars }}
```

Comment:
Where does insert come from?

Reply:
Docs are here: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#insertion

The insert syntax does what the each loop is doing. It adds all the key/value pairs into the map. We also use this in JS (https://github.com/Azure/azure-sdk-for-js/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L49) 

The example from the docs: 
```
# Default values
parameters:
  additionalVariables: {}

jobs:
- job: build
  variables:
    configuration: debug
    arch: x86
    ${{ insert }}: ${{ parameters.additionalVariables }}
  steps:
  - task: msbuild@1
  - task: vstest@2
```